tests/bgp/test_bgp_session_flap.py

```diff
@@ -92,8 +93,11 @@ def setup(tbinfo, nbrhosts, duthosts, enum_frontend_dut_hostname, enum_rand_one_
 
     logger.info("DUT BGP Config: {}".format(duthost.shell("vtysh -n {} -c \"show run bgp\"".format(namespace),
                                                           module_ignore_errors=True)))
-    logger.info("Neighbor BGP Config: {}".format(
-        nbrhosts[tor1]["host"].eos_command(commands=["show run | section bgp"])))
+    if isinstance(nbrhosts[tor1]['host'], EosHost):
```

Comment:
It may not be a good idea to distinguish between different host types within the case.
Better to call the same function here and implement it in each host type file with different commands.

Reply:
Thanks @lipxu. Modified the code to call same function here and implemented in each host file with different commands as suggested. Please take a look.